client/.env.lc

```
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_SEPOLIA_RPC_URL = https://eth-mainnet.g.alchemy.com/v2/MBGDHRRT3_qjtqKzM2DZk
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)